### ISSUE TITLE
Why isn't font-manager in package.json?

### ISSUE THREAD

**Issue by cosmotek** [carloslfu/figma-app-linux#21]
Hey... I was digging into some issues I encountered with the install, related to font-manager install... and I just noticed that the package isn't included in the package.json file, and thus there is no specific version in use. Is there a reason for this? If not, we should find a version that works, and add it to the package.json and package-lock.json in order to make it part of the process.

I realize that the project does lookup font-manager at runtime, is this application expecting a global install? We should find a way to embed this in the app.

**Comment by carloslfu** [comment]
Yes, the reason is that it needs to be installed inside the `app/` folder, from the README: `Go to app/ folder and install font support with npm i font-manager`. I agree it's not the best way and it could be improved.
